docs/blog/connected-registry.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Connected Registry Private Preview
+description: Private preview for ACR connected registry feature.
+ms.topic: post
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
+
+The connected registry feature of ACR allows you to deploy a registry on your premises and synchronize images between the ACR and your premises. It brings the container images and OCI artifacts closer to your on-premise container workloads and increases their performance and reliability.
```

Comment:
@toddysm curious, will we use "on-premises" or "on-premise" phrasing? I see IoT Edge uses "on-premises" https://docs.microsoft.com/en-us/azure/iot-edge/about-iot-edge?view=iotedge-2018-06

Reply:
The wording around "your premises" and on-premises is a bit confusing: @dlepow for assistance here

Reply:
"Your premises" is OK as a noun. Adjective form should be "on-premises", per MSFT style.

Reply:
On-premises is also industry term. It is hard writing content when we remove terms from the language :) I will change to the "on your premises" or "on premises" but I believe we should try to stick with the language the rest of the industry understands.